statistics/handle/handle.go

```diff
@@ -699,3 +704,60 @@ func (h *Handle) statsMetaByTableIDFromStorage(tableID int64, historyStatsExec s
 	count = rows[0].GetInt64(2)
 	return
 }
+
+type statsReader struct {
```

Comment:
add some comment for `statsReader`? Like notifying people that we they should use the same txn?